contrib/opencensus-ext-google-cloud-clientlibs/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '0.2.dev0'
+__version__ = '0.1.3'
```

Comment:
Did you mean to also release this one? This is bumped from the current version of [0.1.2](https://pypi.org/project/opencensus-ext-google-cloud-clientlibs/)

Reply:
Yes, updated description.